### ISSUE TITLE
User can get deleted

### ISSUE THREAD

**Issue by ahaurw01** [SirZach/bbbbbbbbbbbbbbb#29]
Not sure how this happened, but once when I loaded the page (and was already logged in) my user got deleted. Everything else stuck around, like decks and presence, but the user went poof and a new one was created for me. All the relationships were severed. Judging by our `visits` property, looks like that has happened to both of us.

**Comment by SirZach** [comment]
trying out these labels to see if github generates any metrics for us on them. i would like to move all issue tracking to github instead of a combination of github and trello and use trello more collaborative/brainstorming efforts

**Comment by ahaurw01** [comment]
:+1: all for it.

**Comment by ahaurw01** [comment]
Thinking about this bug more- I saw it when I was logged in and transitioning to my decks view. The console barfed some errors about something being in a deleted in-flight state. It's possible this has something to do with the relationship between users and decks. I noticed firebase doing some pretty crazy stuff when you had a relationship defined on both sides. E.g. I think if you manually remove a deck in the firebase UI it will remove its id from the deck array on the user. If we see weird behavior with the relationships, we can always decouple decks from the user model and manually fetch them as needed.
